src/resolver.ts

```diff
@@ -23,3 +42,53 @@ export const resolvers = {
     },
   },
 };
+
+function isThereANumber(str: string) {
+  return isNaN(parseFloat(str));
+}
+
+function isThereALetter(str: string) {
+  const alf = [
+    'a',
+    'b',
+    'c',
+    'd',
+    'e',
+    'f',
+    'g',
+    'h',
+    'i',
+    'j',
+    'k',
+    'l',
+    'm',
+    'n',
+    'o',
+    'p',
+    'q',
+    'r',
+    's',
+    't',
+    'u',
+    'v',
+    'w',
+    'x',
+    'y',
+    'z',
+  ];
+  let def = false;
+  alf.forEach(function (letter) {
+    if (str.toLowerCase().includes(letter)) {
+      def = true;
+    }
+  });
+  return def;
```

Comment:
No Javascript os arrays tem várias funções nativas que ajudam a iterar e fazer alguma lógica sobre algum array. Claro que é possível fazer tudo com `forEach`, mas quando existe outro método que deixa o código mais limpo, é legal tentar utilizá-lo. Aqui, por exemplo, o [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) deve fazer o que vc quer:

```ts
  return alphabet.some((letter) => text.toLowerCase().includes(letter));
```

Mais exemplos: https://www.freecodecamp.org/news/the-javascript-array-handbook/